Packs/CommonScripts/ReleaseNotes/1_11_92.json

```diff
@@ -0,0 +1 @@
+{"breakingChanges":true,"breakingChangesNotes":"The results of GetListRow automation will now be under the context path GetListRow.Results."}
```

Comment:
Please detail the old and new context paths, i.e.:
Changed output from context path .... to context path ...